tests/unit/test_cloned_project.py

```diff
@@ -0,0 +1,67 @@
+from unittest import mock
+
+from ogr.abstract import GitProject
+from packit.cloned_project import ClonedProject
+
+
+def test_cloned_project_name():
+    project = ClonedProject(full_name="namespace/repository_name")
+    assert project.repo == "repository_name"
+    assert project.namespace == "namespace"
+    assert project.full_name == "namespace/repository_name"
+
+
+def test_cloned_project_repo_namespace():
+    project = ClonedProject(repo="repository_name", namespace="namespace")
+    assert project.repo == "repository_name"
+    assert project.namespace == "namespace"
+    assert project.full_name == "namespace/repository_name"
+
+
+@mock.patch.object(GitProject, "get_git_urls", return_value={"git": "ssh url"})
+def test_cloned_project_project(_):
+    project_mock = GitProject(
+        repo="repository_name", namespace="namespace", service=mock.Mock()
+    )
+
+    project = ClonedProject(
+        git_project=project_mock, working_dir=mock.Mock(), git_repo=mock.Mock()
+    )
+    assert project.git_service
+    assert project.git_project
+    assert project.repo == "repository_name"
+    assert project.namespace == "namespace"
+    assert project.full_name == "namespace/repository_name"
+
+
+def test_cloned_project_git_repo():
+    project = ClonedProject(
+        git_repo=mock.Mock(active_branch="branch", working_dir="something"),
+        git_url=mock.Mock(),
+    )
+    assert project.git_repo
+    assert project.working_dir == "something"
+    assert project._branch == "branch"
+
+
+def test_clone_project_checkout_branch():
+    class BranchMock(mock.Mock):
+        checkouted = False
```

Comment:
the word here should be `checked_out` :D

Reply:
Remove that part after moving to flexmock.